### ISSUE TITLE
Add support for Metamako

### ISSUE THREAD

**Issue by gavmckee80** [networklore/nelsnmp#10]
Please see the output from hostinfo.__dict__ below

{'sysobjectid': '1.3.6.1.4.1.43191.1.2.3', 'vendor': 'UNKNOWN', 'description': u'Metamako MOS release 0.13.3 \\(build 5083\\) running on a MetaConnect 48', 'uptime': datetime.timedelta(65, 70212, 490000), 'contact': u'', 'location': u'DC1 ROW 3 CAGE 42', 'version': 'UNKNOWN', 'os': 'UNKNOWN', '_snmp': <nelsnmp.snmp.SnmpHandler object at 0x00000000020B7278>}

Thanks

**Comment by ogenstad** [comment]
Thanks!

I'm not familiar with those devices. Would this be the correct information or should anything be changed?
```
vendor: metamako
os: mos
version: 0.13.3
```
Or is the operating system called something else or should the version include the build or something?

**Comment by gavmckee80** [comment]
Hi Patrick,

I think that looks good.  If you could include the build number with the
version that would be great thanks.

https://www.metamako.com

On Thu, Jul 27, 2017 at 12:33 PM, Patrick Ogenstad <notifications@github.com

**Comment by ogenstad** [comment]
I just pushed a branch called metamako which should fix this. I think it should work, the think I'm not really sure about is how it will escape the \ character around the build.

The output should be:

```
vendor: metamako
os: mos
version: 0.13.3 (build 5083)
```

It would be great if you could test and confirm, otherwise we can fix that part in the future if there's some error.

**Comment by gavmckee80** [comment]
Hi Patrick, 

Thanks for the update.  I can confirm it works.  Please see the output below

In [1]: from nelsnmp.snmp import SnmpHandler
   ...: from nelsnmp.hostinfo.device import HostInfo
   ...:

In [2]: dev = SnmpHandler(host='1.1.1.1', community='public', version='2c')

In [3]: hostinfo = HostInfo(dev)

In [4]: hostinfo.get_all()

In [5]: print hostinfo.vendor
metamako

In [6]: print hostinfo.os
mos

In [7]: print hostinfo.version
0.13.3 (build 5083)

In [8]: print hostinfo.__dict__
{'sysobjectid': '1.3.6.1.4.1.43191.1.2.3', 'vendor': 'metamako', 'description': u'Metamako MOS release 0.13.3 \\(build 5
083\\) running on a MetaConnect 48', 'uptime': datetime.timedelta(125, 47076, 790000), 'contact': u'Operatons', 'locatio
n': u'PRIVATE', 'version': u'0.13.3 (build 5083)', 'os': 'mos', '_snmp': <nelsnmp.snmp.SnmpHandler object at 0x0000000005
8D7470>}

In [9]: print hostinfo.location
PRIVATE

**Comment by ogenstad** [comment]
Thanks! I've released Nelsnmp 0.2.7 where this code is included.
